content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -284,6 +284,37 @@ An encoded Protobuf message with the following IDL:
 
 Clients that receive a response in `application/vnd.kubernetes.protobuf` that does not match the expected prefix should reject the response, as future versions may need to alter the serialization format in an incompatible way and will do so by changing the prefix.
 
+
+## Dry Run
+
+In version 1.12, if the DryRun alpha feature is enabled, the modifying verbs (`POST`, `PUT`, `PATCH`, and `DELETE`) can accept requests in a dry-run mode, which allows users to see if the request would have succeeded (admission chain, validation, merge conflicts, ...) and/or what would have happened without having it actually happen. The response body for the request is as close as possible to a non dry-run response.
```

Comment:
- Possibly clean-up usage of Dry Run,  Dry-run, DryRun
- May want to split up the first sentence.
- Possibly further explain Dry run mode:
Dry run mode helps to evaluate a request through the typical request stages (admission chain, validation, merge conflicts) up until persisting objects to storage.

Reply:
"DryRun" is special because it's the feature flag name, and I used "Dry Run" as the title of the section, but besides that I only used "dry-run" I think. I will switch all the uses to "dry run" so it will be consistent with the title of the section.